examples/showcases/reskinnable-demo/.env.example

```
@@ -81,7 +81,7 @@ INTELLIGENCE_GATEWAY_WS_URL=ws://localhost:7253
 # Verified: an identical user id under a different key resolves to a different
 # memory scope, so the two demos cannot see or delete each other's memories even
 # when pointed at one stack.
-INTELLIGENCE_API_KEY=cpk_s2PRVSED_seed0privat0longtoken01
+CPK_INTELLIGENCE_API_KEY=cpk_s2PRVSED_seed0privat0longtoken01
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CPK_INTELLIGENCE_API_KEY key should go before the INTELLIGENCE_API_URL key

(UnorderedKey)